_layouts/docwithnav.html

```diff
@@ -41,7 +41,7 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    {% if page.no_canonical != true %}<link rel="canonical" href="{{site.url}}{{page.url | replace:page.version,'latest'}}" />{% endif %}
+    <!--{% if page.no_canonical != true %}<link rel="canonical" href="{{site.url}}{{page.url | replace:page.version,'latest'}}" />{% endif %}-->
```

Comment:
you can just remove this instead of commenting it out. We have git history to get it back if we need it.

Reply:
I know....it's just to show for the POC.